.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly add `node_modules` to your `.gitignore` file. This will prevent git from tracking the node modules if you have them from setting up the linters locally.

Reply:
Thanks. I have added the node_modules in the .gitignore file